### PR TITLE
Add proper support for translations

### PR DESCRIPTION
**Pull request by RadxaPanda**
Now all thats missing is the translations for CN which i am unable to do :/

### CODE REVIEW

File: po/radxa-welcome.pot [hunk 1]
```diff
@@ -1,27 +1,27 @@
-# Translations for Radxa's welcome app
-# Copyright (C) 2024 Radxa Computer Co., Ltd
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
 # This file is distributed under the same license as the radxa-welcome package.
-# Radxa Computer Co., Ltd <dev@radxa.com>, 2024.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```
Comment (RadxaYuntian) [inline po/radxa-welcome.pot]: They became placeholder values again.

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]

File: po/radxa-welcome.pot (hunk 1 above)
Comment (RadxaPanda) [reply po/radxa-welcome.pot]: I have no idea how to fix that, it seems it isnt mentioned on meson docs

File: po/radxa-welcome.pot [hunk 2]
```diff
@@ -8,20 +8,20 @@ msgid ""
 msgstr ""
 "Project-Id-Version: radxa-welcome\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2024-07-03 11:19+0300\n"
+"POT-Creation-Date: 2024-07-22 04:08+0300\n"
 "PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
 "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
 "Language-Team: LANGUAGE <LL@li.org>\n"
```
Comment (RadxaYuntian) [inline po/radxa-welcome.pot:14]: More placeholder values.

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]
Fix those then you can squash the commit

File: po/radxa-welcome.pot (hunk 2 above)
Comment (RadxaPanda) [reply po/radxa-welcome.pot:14]: Those are actually meant to be placeholder values

Comment (RadxaPanda) [reply po/radxa-welcome.pot:14]: Since the pot isn't a translation for any language it has placeholders take a look at zh_cn.po it autofilled there

**Review by RadxaYuntian — APPROVED** [review]

**Pull request merged**